conf/messages.en

```diff
@@ -437,7 +437,7 @@ doYouHaveAnEmailAddress.title = Do you have an email address that you are happy
 doYouHaveAnEmailAddress.heading = Do you have an email address that you are happy to be contacted on by HMRC?
 doYouHaveAnEmailAddress.checkYourAnswersLabel = Do you have an email address that you are happy to be contacted on by HMRC?
 doYouHaveAnEmailAddress.paragraph = HMRC will only use this email address if they need to contact you about this disclosure
-doYouHaveAnEmailAddress.error.required = Select yes if you have an email address that HMRC can contact you with
+doYouHaveAnEmailAddress.error.required = Select yes if you have an email address that HMRC can contact you with.
```

Comment:
Just one comment, we need this to not have a full stop

Reply:
fixed!!